content/articles/data-transformation-with-linq/index.md

```diff
@@ -0,0 +1,181 @@
+
+### Introduction
+LINQ (Language-Integrated Query) is used for more than just data retrieval. It's also a useful tool for data transformation. New output may be gotten using LINQ by capturing source sequences and converting them in various ways.
+
+Grouping and sorting are techniques that usually change the sequence of a query, but the same change can be achieved without using those techniques. This is achievable when a learner or a programmer uses the `select` clause. For example, a programmer may perform or carry out some of the following tasks.
+1. In creating one output query consisting of another type, a programmer can combine several input queries.
+2. A programmer may still opt to take a few elements from each element in the source sequence to create an output sequence.
+3. By putting into consideration the results of operations on source data and several other things, an output sequence can still be obtained.
+
+### Table of contents
+ - Creating a Single Output Sequence with Multiple Inputs 
+- Subsets of each Source Element are chosen
+- Operations that are performed on source elements
+- In-memory objects - XML transformation
+- Conclusion
+-  What next
+### Creating a Single Output Sequence with Multiple Inputs
+When working with LINQ in the implementation of various concepts, one of the capabilities of LINQ in the field of data transformation is that it can accept more than one input and produce a single output sequence. The description in the example below explains how we can bring together more than one in-memory data structure. Consider the two sorts of classes listed below.
+ 
+```c#
+class Employer
+{
+    public string FirstName{(set; get;)}
+    public string LastName{(set; get;)}
+    public int JobNo{(set; get;)}
+    public string city{(set; get;)}
+}
+class Employee
+{
+     public string FirstName{(set; get;)}
+    public string LastName{(set; get;)}
+    public int JobNo{(set; get;)}
+    public string city{(set; get;)}
+    
+}
+```

```

Comment:
I also feel the code snippets are not well explained. make your article flow as much as possible. Grab as much attention of the reader as possible!